Rakefile

```
@@ -27,7 +27,7 @@ RuboCop::RakeTask.new(:internal_investigation)
 # The `ascii_spec` task has not been failing for a while, so it will not be run by default.
 # However, `ascii_spec` task will continue to be checked in CI. If there are any failures
 # originating from `ascii_spec` in CI, please run `bundle exec ascii_spec` to investigate.
-task default: %i[documentation_syntax_check spec prism_spec internal_investigation]
+task default: %i[documentation_syntax_check spec prism_spec internal_investigation codespell]
```

Since running `codespell` should be fast, I think it makes sense to move it before running tests, so that if there's a failure you don't have to wait for 2 test runs and rubocop to complete.

```suggestion
task default: %i[documentation_syntax_check codespell spec prism_spec internal_investigation]
```

---

I moved it all the way to the front :+1:, it takes about 100ms for me. I don't think the doc check _needs_ to happen before and that is also not the fastest thing to do